test/testcafe/spec/TerminalView_spec.js

```diff
@@ -69,6 +71,8 @@ fixture('Terminal view').meta('v3', true);
 async function setup(t) {
   const terminalPageObject = userVariables.v3 ? new TerminalPageObjectV3(t) : new TerminalPageObject(t);
   await terminalPageObject.navigateToPage();
+  // ensure form has loaded
+  await t.expect(Selector('form').exists).eql(true);
```

Comment:
@leemchale-okta FYI I moved this again to _just_ the TerminalView spec for now. We can re-evaluate if we should move it back up with better guards around it in the future if we see the form loading flakiness again in other specs.

Reply:
okay sounds good